social-network/src/Article.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import Image from './Image.js';
+import Text from './Text.js';
+import AuthorBar from './AuthorBar.js';
+import Bookmark from './Bookmark.js';
+
+class Article extends React.Component {
+
+  static propTypes = {
+    item: PropTypes.object,
+    link: PropTypes.string,
+    isYourArticles: PropTypes.bool,
+    idx: PropTypes.number
+  }
```

Comment:
It looks like item is the only direct proptype and the rest are keys of that object. In this case consider using Proptypes.shape({}) and placing your keys in there. You can review Class 3 for more examples of how to define Proptypes.